### PR TITLE
Feature/ARCWELL 463 bulk import 

### PR DESCRIPTION
**Pull request by mberlin-arcweb**
adding the admin and server pieces to allow for bulk import

### CODE REVIEW

File: packages/server/app/controllers/events_controller.ts [hunk 1]
```diff
@@ -180,4 +183,22 @@ export default class EventsController {
     await event.delete()
     response.status(204).send('')
   }
+
+  async bulk({ request, response }: HttpContext) {
+    await request.validateUsing(bulkUploadValidator)
+    const file = request.file('file')
+    const tpyeKeyData = request.only(['typeKey'])
```
Comment (tmolumby) [inline packages/server/app/controllers/events_controller.ts]: Typo?

File: packages/server/app/controllers/facts_controller.ts [hunk 2]
```diff
@@ -181,4 +184,22 @@ export default class FactsController {
     await fact.delete()
     response.status(204).send('')
   }
+
+  async bulk({ request, response }: HttpContext) {
+    await request.validateUsing(bulkUploadValidator)
+    const file = request.file('file')
+    const tpyeKeyData = request.only(['typeKey'])
```
Comment (tmolumby) [inline packages/server/app/controllers/facts_controller.ts]: Typo again?

File: packages/admin/src/assets/examples/events_appointment.csv [hunk 3]
```diff
@@ -0,0 +1 @@
+startedAt,endedAt,personId,resourceId
```
Comment (timgetz) [inline packages/admin/src/assets/examples/events_appointment.csv:1]: What is the purpose of all these csv files with only headers?
Also, being in the asset folder will mean these get deployed.  Is that intended?

**Review by timgetz — CHANGES_REQUESTED** [review]

File: packages/admin/src/assets/examples/events_appointment.csv (hunk 3 above)
Comment (mberlin-arcweb) [reply packages/admin/src/assets/examples/events_appointment.csv:1]: these are the sample file that get downloaded when you press download example. they are going to be needed in the app.

File: packages/server/app/controllers/events_controller.ts (hunk 1 above)
Comment (mberlin-arcweb) [reply packages/server/app/controllers/events_controller.ts]: fixed

File: packages/server/app/controllers/facts_controller.ts (hunk 2 above)
Comment (mberlin-arcweb) [reply packages/server/app/controllers/facts_controller.ts]: fixed

File: packages/admin/src/assets/examples/events_appointment.csv (hunk 3 above)
Comment (timgetz) [reply packages/admin/src/assets/examples/events_appointment.csv:1]: I would prefer these get generated, so we can make them dynamic in the future.  We will want to support dimension columns which will be data based.  If we want to keep them for now, that's fine with me.  Do we have a story to support importing with dimensions?  If, so I'd like dynamic file generations be a requirement in that.  If not, would you make one or update a confluence page.

Comment (mberlin-arcweb) [reply packages/admin/src/assets/examples/events_appointment.csv:1]: i agree, i think the fianl version should have a generated solution, i will add that info to the ticket or make a new one.

**Review by timgetz — APPROVED** [review]

**Pull request merged**